### PR TITLE
Use category ME instead of [merge] in PR title to allow merge commit

### PR DESCRIPTION
**Pull request by atomiix**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | This PR aims to use the "Category?" section in the PR description table to allow merge commit only when the category is ME instead of adding [merge] in the PR title
| Type?             | improvement
| Category?         | PM
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| How to test?      | CI is green
| Possible impacts? | 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/28007)

### CODE REVIEW

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -15,7 +14,15 @@ jobs:
         with:
           access_token: ${{ github.token }}
 
+      - name: Check is merge PR
+        id: check-is-merge
+        run: |
+          if [[ """${{ github.event.pull_request.body }}""" =~ \|?[[:space:]]*Category\?[[:space:]]*\|[[:space:]]*ME[[:space:]]+ ]]; then
```
Comment (matks) [inline .github/workflows/pr.yml:20]: I have no idea what this is doing 🤣

Comment (atomiix) [reply .github/workflows/pr.yml:20]: That's a bash regex equivalent of the regex here: https://github.com/PrestaShop/prestonbot/blob/master/src/AppBundle/PullRequests/BodyParser.php#L12 with "Category" et "ME" as %s arguments.

**Review by Progi1984 — APPROVED** [review]

File: .github/workflows/pr.yml (hunk 1 above)
Comment (MeKeyCool) [reply .github/workflows/pr.yml:20]: I like your `with "Category" et "ME"` xD My brain is melting thanks to you xD

**Review by MeKeyCool — COMMENTED** [review]
I don't like `PM` category for this kind of modification.

Imho it's more about devops or CI/CD but according https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/pull-requests/#the-pull-request-form-1 there is no such category code.

**Review by matthieu-rolland — APPROVED** [review]

**Comment by matthieu-rolland** [comment]
thank you @atomiix  !

**Pull request merged**